kubernetes/README.md

```diff
@@ -73,20 +91,55 @@ Once the DNS record has propagated, you can test your instance by requesting `ht
 You can do this with `curl`, as an example:
 
 ```sh
-curl --header "Authorization: Bearer <bearertoken>" https://<domain>/scim/Users
+curl --header "Authorization: Bearer TOKEN_GOES_HERE" https://<domain>/scim/Users
 ```
 
 You can now continue with the administration guide to configure your Identity Provider to enable provisioning with your SCIM Bridge.
 
 
+## Upgrading
+
+It should be relatively simple to upgrade the SCIM Bridge from previous versions.
+
+First, you `git pull` the latest versions from this repository. Then, you re-apply the `.yaml` files.
+
+```bash
+cd scim-examples/
+git pull
+cd kubernetes/
+kubectl apply -f .
+```
+
+This should seamlessly upgrade your SCIM Bridge to the latest version. The process takes about 2-3 minutes for the Bridge to come back online.
+
+**NOTE:** As of October 2020, the `scim-examples` Kubernetes deployment now uses `op-scim-config.yaml` to set the configuration needed for your SCIM Bridge, and has changed the deployment names from `op-scim` to `op-scim-bridge`, and `redis` to `op-scim-redis` for clarity and consistency. 
+
+You’ll need to re-configure your options in `op-scim-config.yaml`, particularly `OP_LETSENCRYPT_DOMAIN`. You may also want to delete your previous `op-scim` and `redis` deployments to prevent conflict between the two versions.
+
+```bash
+kubectl delete deployments.apps op-scim redis
```

Comment:
This is the preferred k8s-cli as it makes it clear that it delete both deployments 

```suggestion
kubectl delete deployment/opscim deployment/redis
```